src/Cosmos/JsonSerialization.h

```diff
@@ -14,6 +14,7 @@ using string = std::string;
 using json = nlohmann::json;
 
 extern const string TYPE_PREFIX_MSG_SEND;
+extern const string TYPE_PREFIX_MSG_TRANSFER;
```

Comment:
seems not used?

Reply:
Correct, that was for Json only, cleaned up

Reply:
this line is still here